amqp/src/main/scala/akka/stream/alpakka/amqp/javadsl/AmqpRpcFlow.scala

```diff
@@ -0,0 +1,30 @@
+package akka.stream.alpakka.amqp.javadsl
+
+import akka.NotUsed
+import akka.stream.alpakka.amqp._
+import akka.stream.javadsl.{Flow, Source}
+import akka.util.ByteString
+
+import scala.concurrent.Future
+
+object AmqpRpcFlow {
+
+  /**
+    * Java API: Creates an [[AmqpRpcFlow]] with given settings and buffer size.
+    */
+  def create(settings: AmqpSinkSettings, bufferSize: Int) =
```

Comment:
Sounds good. I've always been awful at writing docs. I'll give it a shot though.